Ducky-Chickens/bangazon-site
# User can view status of products uploaded

# Description
Add ability to view products, how many have been sold, and how many are left that the current user is selling.

## Related Ticket(s)
Completes Issue #7 (User can view status of products uploaded)

## Expected Behavior
When you navigate to /inventory, the current user should see all products being sold.  When you navigate to it without being logged in, it should redirect you to the login page.  You should only see the Inventory link on the nav when you are logged in.

## Steps to Test Solution
1. Run `node db-build.js`
1. Run `nodemon app.js`
1. When you navigate to `localhost:8080/inventory` when you are not logged in, you should be led to a login page.  
1. When not logged in, you should NOT be able to see the Inventory link on the nav bar.
1. `After logging` in, when you go to `localhost:8080/inventory`, you should see a list of items and how many are left / have been sold, per the example below for user #1.  (The first and second values are labeled by html classes.  These should be styled later.)
```
Inventory
Awesome Soft Cheese
0
1
Incredible Rubber Car
1
0
Licensed Concrete Computer
1
0
Sleek Wooden Towels
2
0
Handcrafted Steel Pizza
2
0
Awesome Rubber Chips
4
1
```

## Documentation
